docs/css/issues-list.css

```diff
@@ -0,0 +1,88 @@
+#issues-list {
+	padding: 0;
+}
+
+#issues-list .issue {
+	border-bottom: 5px solid var(--color-bg-page);
+}
+
+	.issue .x2{
+		border-left: 10px solid;
+		;
+	}
+
+	.issue .issue-description, 
+	.issue .issue-labels {
+		padding-top: calc(var(--unit-gap) / 3); 
+		padding-bottom: calc(var(--unit-gap) / 3);
+	}
+
+	.issue .issue-description {
+		padding-left: var(--unit-gap);
+		font-size: 1.1em;
+	}
+
+		.issue .issue-description * {
+			line-height: 1.1em;
+		}
+
+		.issue-description .issue-number {
+			font-weight: 700;
+		}
+
+		.issue-description .issue-url {
+			text-decoration: underline;
+		}
+
+		.issue-description .issue-type {
```

Comment:
empty class

Reply:
keeping it for now as we might get a different style in here.